proto/zigbeealliance/distributedcomplianceledger/pki/query.proto

```diff
@@ -29,343 +33,439 @@ option go_package = "github.com/zigbee-alliance/distributed-compliance-ledger/x/
+  }
+
+  // Queries a ApprovedRootCertificates by index.
+  rpc ApprovedRootCertificates(QueryGetApprovedRootCertificatesRequest) returns (QueryGetApprovedRootCertificatesResponse) {
+    option (google.api.http).get = "/dcl/pki/root-certificates";
+  }
+  // Queries a RevokedRootCertificates by index.
+  rpc RevokedRootCertificates(QueryGetRevokedRootCertificatesRequest) returns (QueryGetRevokedRootCertificatesResponse) {
+    option (google.api.http).get = "/dcl/pki/revoked-root-certificates";
+  }
+
+
+  // Queries a RejectedCertificate by index.
+  rpc RejectedCertificate(QueryGetRejectedCertificatesRequest) returns (QueryGetRejectedCertificatesResponse) {
+    option (google.api.http).get = "/dcl/pki/rejected-certificates/{subject}/{subjectKeyId}";
+  }
+
+  // Queries a list of RejectedCertificate items.
+  rpc RejectedCertificateAll(QueryAllRejectedCertificatesRequest) returns (QueryAllRejectedCertificatesResponse) {
+    option (google.api.http).get = "/dcl/pki/rejected-certificates";
+  }
+
+  // Queries a PkiRevocationDistributionPoint by index.
+  rpc PkiRevocationDistributionPoint(QueryGetPkiRevocationDistributionPointRequest) returns (QueryGetPkiRevocationDistributionPointResponse) {
+    option (google.api.http).get = "/dcl/pki/revocation-points/{issuerSubjectKeyID}/{vid}/{label}";
+  }
+
+  // Queries a list of PkiRevocationDistributionPoint items.
+  rpc PkiRevocationDistributionPointAll(QueryAllPkiRevocationDistributionPointRequest) returns (QueryAllPkiRevocationDistributionPointResponse) {
+    option (google.api.http).get = "/dcl/pki/revocation-points";
+  }
+
+  // Queries a PkiRevocationDistributionPointsByIssuerSubjectKeyID by index.
+  rpc PkiRevocationDistributionPointsByIssuerSubjectKeyID(QueryGetPkiRevocationDistributionPointsByIssuerSubjectKeyIDRequest) returns (QueryGetPkiRevocationDistributionPointsByIssuerSubjectKeyIDResponse) {
+    option (google.api.http).get = "/dcl/pki/revocation-points/{issuerSubjectKeyID}";
+  }
+
+  // Queries a list of NocCertificates items.
+  rpc NocCertificatesAll(QueryNocCertificatesRequest) returns (QueryNocCertificatesResponse) {
+    option (google.api.http).get = "/dcl/pki/all-noc-certificates";
+  }
+  // Queries a NocCertificatesBySubject by index.
+  rpc NocCertificatesBySubject(QueryGetNocCertificatesBySubjectRequest) returns (QueryGetNocCertificatesBySubjectResponse) {
+    option (google.api.http).get = "/dcl/pki/all-noc-certificates/{subject}";
+  }
+  // Queries a NocCertificates by index.
+  rpc NocCertificates(QueryGetNocCertificatesRequest) returns (QueryGetNocCertificatesResponse) {
+    option (google.api.http).get = "/dcl/pki/all-noc-certificates/{subject}/{subjectKeyId}";
+  }
+
+  // Queries a NocCertificatesByVidAndSkid by index.
+  rpc NocCertificatesByVidAndSkid(QueryGetNocCertificatesByVidAndSkidRequest) returns (QueryGetNocCertificatesByVidAndSkidResponse) {
+    option (google.api.http).get = "/dcl/pki/noc-certificates/{vid}/{subjectKeyId}";
```

Comment:
Let's consider renaming 
- `noc-certificates` to `noc-vid-certificates`
- `noc-root-certificates` to `noc-vid-root-certificates`
- `noc-ica-certificates` to `noc-vid-ica-certificates`

Reply:
Fixed